storage-hbase/pom.xml

```diff
@@ -157,6 +157,7 @@
                                     <include>org.apache.kylin:kylin-core-cube</include>
                                     <include>org.roaringbitmap:RoaringBitmap</include>
                                     <include>com.tdunning:t-digest</include>
+                                    <include>com.tdunning:t-digest</include>
```

Comment:
Can we delete this line?

Lines 159 and 160 are the same: `<include>com.tdunning:t-digest</include>`